webcompat/static/js/lib/bugform.js

```diff
@@ -111,32 +108,29 @@ function BugForm() {
     var canvas = document.createElement("canvas");
     var ctx = canvas.getContext("2d");
 
-    img.onload = _.bind(
-      function() {
-        // scale the tmp canvas to 50%
-        canvas.width = Math.floor(img.width / 2);
-        canvas.height = Math.floor(img.height / 2);
-        ctx.scale(0.5, 0.5);
-        // draw back in the screenshot (at 50% scale)
-        // and re-serialize to data URI
-        ctx.drawImage(img, 0, 0);
-        // Note: this will convert GIFs to JPEG, which breaks
-        // animated GIFs. However, this only will happen if they
-        // were above the upload limit size. So... sorry?
-        var screenshotData = canvas.toDataURL("image/jpeg", 0.8);
-
-        // The limit is 4MB (which is crazy big!), so let the user know if their
-        // file is unreasonably large at this point (after 1 round of downsampling)
-        if (screenshotData > this.UPLOAD_LIMIT) {
-          this.makeInvalid("image", { altHelp: true });
-          return;
-        }
+    img.onload = _.bind(function() {
+      // scale the tmp canvas to 50%
+      canvas.width = Math.floor(img.width / 2);
+      canvas.height = Math.floor(img.height / 2);
+      ctx.scale(0.5, 0.5);
+      // draw back in the screenshot (at 50% scale)
+      // and re-serialize to data URI
+      ctx.drawImage(img, 0, 0);
+      // Note: this will convert GIFs to JPEG, which breaks
+      // animated GIFs. However, this only will happen if they
+      // were above the upload limit size. So... sorry?
+      var screenshotData = canvas.toDataURL("image/jpeg", 0.8);
+
+      // The limit is 4MB (which is crazy big!), so let the user know if their
+      // file is unreasonably large at this point (after 1 round of downsampling)
+      if (screenshotData > this.UPLOAD_LIMIT) {
+        this.makeInvalid("image", { altHelp: true });
+        return;
+      }
 
-        this.addPreviewBackgroundAndUpload(screenshotData);
-        (img = null), (canvas = null);
-      },
-      this
-    );
+      this.addPreviewBackgroundAndUpload(screenshotData);
+      (img = null), (canvas = null);
+    }, this);
```

Comment:
This change makes me very happy, before it was too many line breaks/whitespace. 👍